third_party/rust/Cargo.toml

```diff
@@ -51,54 +53,90 @@ serde_json = "1"
 sha2 = "0.9"
 small_ctor = "0.1"
 static_assertions = "1"
-strobe-rs = "0.8"
+strobe-rs = "0.8.1"
 tempfile = "3"
-thiserror = "1"
-url = "2"
-urlencoding = "1"
+thiserror = "^1.0.30"
+url = "2.2"
+urlencoding = "1.1.1"
 wycheproof = "0.4"
-zeroize = "1"
 
 [dependencies.adblock]
 default-features = false
-version = "0.7"
+version = "0.7.9"
 features = ["full-regex-handling", "debug-info", "css-validation"]
 
 [dependencies.bigdecimal]
 version = "0.1"
 features = ["serde"]
 
+[dependencies.bitvec]
+default-features = false
+version = "1.0.1"
+
 [dependencies.bls-signatures]
 default-features = false
 version = "0.12"
 features = ["pairing"]
 
+[dependencies.byteorder]
+default-features = false
```

Comment:
I'm surprised this works; a lot of crates use Read/WriteBytesExt even for parsing buffers. But if it compiles, ok for now.

Reply:
See above, this was produced by `gnrt gen` so it should be based on the features listed in the crates that depend on it. Where are you seeing that it is needed without `default-features = false`?

Reply:
it does seem like it should have `i128` from `curve25519-dalek` and `std` from `rmp-serde`, `fxhash`, etc... and those do appear in the `BUILD.gn`

```
features = [
    "i128",
    "std",
  ]
```